MaxText/layers/models.py

```diff
@@ -245,11 +272,13 @@ def __call__(self,
             y,
             decoder_segment_ids,
             decoder_positions,
+            padding_mask,
             deterministic,
             model_mode,
         )
 
-    y = RMSNorm(dtype=cfg.dtype, name='decoder_norm', epsilon=cfg.rms_norm_epsilon,kernel_axes=('embed',))(y)
+    norm_layer = self.get_norm_layer()
```

Comment:
Probably just put this on one line? 

self.get_norm_layer()(dtype=cfg.dtype, name='decoder_norm', epsilon=cfg.norm_epsilon, kernel_axes=('embed',))(y)

Reply:
Good advice!